### PR TITLE
css-layout-classes - James Yang

### PR DESCRIPTION
**Pull request by jamesdae**
Screen Shot 2022-10-13 at 3 19 14 PM

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS Layout Classes</title>
+</head>
+<body>
+  <div class="container">
+    <div class="row"> <!--First row for h1-->
+      <h1>Choose your character!</h1>
```
Comment (Zhiwutian) [inline css-layout-classes/index.html:10]: make sure to wrap this in a column-full class and give it the correct flex basis

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looks great! One small update!

**Comment by jamesdae** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**